src/components/common/Actions.vue

```diff
@@ -308,82 +320,81 @@ export default class Actions extends Mixins(
       }
     }
 
-    if (this.isFilingValid) {
-      // prevent double saving
-      if (this.isBusySaving) return
-      // If File and Pay is successful, setIsFilingPaying shouldn't be reset to false.
-      // This prevent buttons from being re-enabled if the page is slow to redirect.
-      this.setIsFilingPaying(true)
-
-      if (
-        this.getEffectiveDateTime.effectiveDate &&
-        !this.isValidDateTime(this.getEffectiveDateTime.effectiveDate)
-      ) {
-        this.setEffectiveDateTimeValid(false)
-
-        window.scrollTo({ top: 1250, behavior: 'smooth' })
-        this.setIsFilingPaying(false)
-        return
-      }
-
-      // If a NR was used, validate it before filing submission.
-      if (this.getNameRequestNumber) {
-        try {
-          await this._validateNameRequest(
-            this.getNameRequestNumber,
-            this.getNameRequestApplicant.phoneNumber,
-            this.getNameRequestApplicant.emailAddress
-          )
-        } catch (error) {
-          console.log('Error validating NR in onClickFilePay():', error) // eslint-disable-line no-console
-          this.setIsFilingPaying(false)
-          return
-        }
+    if (!this.isFilingValid) {
+      if (!this.isContinuationInAuthorization) {
+        // if this is a Continuation In Authorization then let that page scroll as needed
+        // otherwise, smooth-scroll to the top of the page
+        window.scrollTo({ top: 0, behavior: 'smooth' })
       }
+      this.setIsFilingPaying(false)
+      return
+    }
```

Comment:
Moving this block here (was previously at bottom) simplifies the logic.

The main thing is to ensure the flag is reset to false if we're not redirecting.